build/generate/functionTypes.go

```diff
@@ -203,7 +203,7 @@ func (f Function) docs() string {
 	if f.Deprecated {
 		content += "\n\n@deprecated"
 	}
-	content += "\n\n@see https://dnscontrol.org/js#" + f.Name
+	content += "\n\n@see https://docs.dnscontrol.org/language-reference/domain-modifiers/" + strings.ToLower(f.Name)
```

Comment:
@j-f1 do you have any suggestions on how to solve the subpaths in a neat way? Based on a directory detection doesn't sound like a very solid solution to me. Especially considering @tlimoncelli's desire to continue splitting the documentation. See https://github.com/StackExchange/dnscontrol/pull/1809#issuecomment-1377431010. An extra `frontMatter` parameter then? What do you think?

Reply:
You could store `t.Name()` in the `Function` strict (maybe as `type` or something else?) and then inject it into the URL?

Reply:
I don't understsand the question :-(

Reply:
I will explain the problem in more detail. Before the introduction of Gitbook, all JavaScript DSL URLs were the same because all information was on the same page.

Example URL of `Domain Modifiers` > `A`
https://stackexchange.github.io/dnscontrol/js#A

Example URL of `Top Level Functions` > `D`
https://stackexchange.github.io/dnscontrol/js#D

Example URL of `Record Modifiers` > `CAA_BUILDER`
https://stackexchange.github.io/dnscontrol/js#CAA_BUILDER

The same examples with the new Gitbook (SEO) URLs:
- https://docs.dnscontrol.org/language-reference/domain-modifiers/a
- https://docs.dnscontrol.org/language-reference/top-level-functions/d
- https://docs.dnscontrol.org/language-reference/record-modifiers/caa_builder

Where 'a mapping' must take place between the directory and the new SEO URLs. Since the SEO URLs are based on the Gitbook menu title, I was unsure whether the directory names are the right choice for this mapping to take place or whether this must be a configuration within the Markdown file (duplicate content). 

Based on @j-f1's suggestion, I stick with the idea of mapping the directories back to the SEO URLs. After all, it is and remains a deep link URL. No big deal. 🤷‍♂️